core/domain/exp_services_test.py

```diff
@@ -629,14 +793,9 @@ def test_loading_and_validation_and_deletion_of_demo_explorations(self):
         for exp_id in demo_exploration_ids:
             start_time = datetime.datetime.utcnow()
 
-            with self.swap(
-                html_validation_service, 'get_filename_with_dimensions',
-                mock_get_filename_with_dimensions):
-                exp_services.load_demo(exp_id)
```

Comment:
have we made `load_demo` independent of `html_validation_service`?

Reply:
Yes that is correct

Reply:
Whether `load_demo` was dependent on  `html_validation_service` before? If yes can you share the line of code?

Reply:
Hmm I cannot find when load_demo was dependent on html_validation_service but my guess is it must be or else the line wouldn't be there. In our current codebase load_demo is not dependent on html_validation_service and this is why this line is untested(i.e its not even being called)

Reply:
I am not sure about this one. `get_filename_with_dimensions` was added by @ishucr7, I think during last GSoC for obtaining the filename of an image file with the height and width dimensions, so I am not sure how load_demo was dependent on the function. I think #6989 updated some of the filenames to match the image filenames regex so we don't need to mock the `get_filename_with_dimensions` function for loading the demo exps.